jasmine/defs.bzl

```diff
@@ -0,0 +1,27 @@
+"Public API re-exports"
+
+
+load("//jasmine/private:jasmine_test.bzl", "lib")
+load("@aspect_rules_js//js:libs.bzl", "js_binary_lib")
+
+_jasmine_test = rule(
+    doc = """The Jasmine rule run tests under the Jasmine framework with Bazel.""",
```

Comment:
"Run tests under the Jasmine framework"?

Reply:
fixed it.